File: front/src/interfaces/MailLogDto.ts
export interface MailLogDto {
  id?: string | null;
  refApplication?: string | null;
  fromAddress?: string | null;
  toAddresses?: string | null;
  ccAddresses?: string | null;
  bccAddresses?: string | null;
  subject?: string | null;
  body?: string | null;
  attachmentsJson?: string | null;
  status?: string | null;
  errorMessage?: string | null;
  fingerprint?: string | null;
  isPatched?: boolean | null;
  sentAtUtc?: string | null;
  occurrences?: number | null;
}

File: api/DTOs/Mail/MailLogDto.cs
namespace api.DTOs.Mail
{
    public sealed class MailLogDto
    {
        public Guid? Id { get; set; }
        public Guid? RefApplication { get; set; }
        public string? FromAddress { get; set; }
        public string? ToAddresses { get; set; }
        public string? CcAddresses { get; set; }
        public string? BccAddresses { get; set; }
        public string? Subject { get; set; }
        public string? Body { get; set; }
        public string? AttachmentsJson { get; set; }
        public string? Status { get; set; }
        public string? ErrorMessage { get; set; }
        public string? Fingerprint { get; set; }
        public bool? IsPatched { get; set; }
        public DateTime? SentAtUtc { get; set; }
        public long? Occurrences { get; set; }
    }
}
